build.xml

```diff
@@ -578,7 +578,7 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
             <fileset dir="${java.src.dir}"/>
             <fileset dir="${src_generated.dir}" excludes="**/.generated"/>
             <manifest>
-                <attribute name="Main-Class" value="org.apache.zookeeper.server.quorum.QuorumPeer" />
+                <attribute name="Main-Class" value="org.apache.zookeeper.server.quorum.QuorumPeerMain" />
```

Comment:
does this change also need to be made on https://github.com/apache/zookeeper/pull/323/files#diff-2cccd7bf48b7a9cc113ff564acd802a8R622 (line 622)?